roles/core/nfs_server/readme.rst

```diff
@@ -60,6 +60,7 @@ Packages installed:
 Changelog
 ^^^^^^^^^
 
+* 1.0.4: Added molecule tests. osmocl <osmocl@osmo.cl>
```

Comment:
Please update the changelog message to reflect the change for end-users (new variable)

Reply:
Changelog updated in commit : 4dd4ed5